cmd/kpod/load.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	dockerArchive = "docker-archive:"
+	oci           = "oci:"
+)
+
+type loadOptions struct {
+	oci   bool
+	input string
+	quiet bool
+}
+
+var (
+	loadFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "oci",
```

Comment:
What if the user has saved an image with the same name to docker-archive and oci, but wants to load the docker-archive one?